resources/wikia/ui_components/modal/css/modal_default.scss

```diff
@@ -209,10 +209,7 @@ $large-modal-margin: 50px;
 
 // styles required to disable scrolling behind modal
 body.with-blackout {
-
 	&.background-dynamic {
-		position: static;
```

Comment:
`&.background-`**`dynamic`** {
`		position: `**`static`**`;`

That didn't sound to dynamic to me. :grin: